### PR TITLE
Fix log limits

### PR DESCRIPTION
**Pull request by michaeladda**
unfortunately only tested manually

### CODE REVIEW

File: cli/src/utils/cli-flags.ts [hunk 1]
```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
...
+    ...options,
     parse(val) {
+      const { min, max } = options;
       if (!isInt(val, { min, max })) {
         error(`Expected an integer between ${min} and ${max} but received: ${val}`);
       }
       return parseInt(val, 10);
     },
-  }),
+  })(),
```
Comment (arielshaqed) [inline cli/src/utils/cli-flags.ts]: I don't understand this.  It looks like you're calling the return value of `flags.build` as a function with no args, and using the return of _that_ function as the value of `minMaxInt`.  How does this even work (or compile, assuming reasonable types)?

Or, why do you need to call the return value from `flags.build` here, but not below in `durationOrISO8601`?

Comment (arielshaqed) [reply cli/src/utils/cli-flags.ts]: Yuck.  Thanks for explaining!

File: cli/src/utils/cli-flags.ts [hunk 2]
```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
...
+export type MinMaxIntFlagOptions = Parameters<(typeof flags.integer)>[0] & Partial<{
   min: number,
   max: number,
-};
+}>;
 
 export default {
   ...flags,
-  minMaxInt: ({ min, max }: MinMaxIntFlagOptions) => flags.integer({
+  minMaxInt: (options: MinMaxIntFlagOptions) => flags.build({
```
Comment (arikmaor) [inline cli/src/utils/cli-flags.ts]: why change this?

File: cli/src/utils/cli-flags.ts [hunk 3]
```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
 import { isISO8601, isInt } from 'validator';
 import ms from 'ms';
 
-export type MinMaxIntFlagOptions = Parameters<(typeof flags.integer)>[0] & {
+export type MinMaxIntFlagOptions = Parameters<(typeof flags.integer)>[0] & Partial<{
```
Comment (arikmaor) [inline cli/src/utils/cli-flags.ts]: If your'e not calling `flags.integer` I guess `Parameters<(typeof flags.integer)>[0]` should be changed also.
BUT, I don't understand why you're changing the code here at all...

File: cli/src/utils/cli-flags.ts [hunk 4]
```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
...
-};
+}>;
 
 export default {
   ...flags,
-  minMaxInt: ({ min, max }: MinMaxIntFlagOptions) => flags.integer({
+  minMaxInt: (options: MinMaxIntFlagOptions) => flags.build({
+    ...options,
     parse(val) {
+      const { min, max } = options;
```
Comment (arikmaor) [inline cli/src/utils/cli-flags.ts]:
```suggestion
```

File: cli/src/utils/cli-flags.ts (hunk 2 above)
Comment (arikmaor) [reply cli/src/utils/cli-flags.ts]:
```suggestion
  minMaxInt: ({ min, max, ...options }: MinMaxIntFlagOptions) => flags.integer({
```

File: cli/src/utils/cli-flags.ts [hunk 5]
```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
...
   min: number,
   max: number,
-};
+}>;
 
 export default {
   ...flags,
-  minMaxInt: ({ min, max }: MinMaxIntFlagOptions) => flags.integer({
+  minMaxInt: (options: MinMaxIntFlagOptions) => flags.build({
+    ...options,
```
Comment (arikmaor) [inline cli/src/utils/cli-flags.ts]: consider moving after `parse`

File: cli/src/utils/cli-flags.ts (hunk 1 above)
Comment (arikmaor) [inline cli/src/utils/cli-flags.ts]:
```suggestion
  }),
```

File: cli/src/utils/cli-flags.ts (hunk 2 above)
Comment (michaeladda) [reply cli/src/utils/cli-flags.ts]: the default is an important option, and without using build the default does not take place

File: cli/src/utils/cli-flags.ts (hunk 1 above)
Comment (michaeladda) [reply cli/src/utils/cli-flags.ts]: sorry, won't fly. build needs an immediate call (integer doesn't of course). See my reply above.

**Pull request merged**